src/app/projects/projects.controller.js

```diff
@@ -5,7 +6,183 @@
     .module('gcloudConsole')
     .controller('ProjectsCtrl', ProjectsCtrl);
 
-  function ProjectsCtrl($scope, $projects) {
-    $scope.$projects = $projects;
+  /* @ngInject */
+  function ProjectsCtrl($rootScope, $projects, $mdDialog, $mdToast, $firebaseUtils) {
+    var projects = this;
+
+    projects.map = $projects.projects;
+    for (var projectId in projects.map) {
```

Comment:
This is mostly a nitpick but, we could maybe clean this up with [`angular.forEach`](https://docs.angularjs.org/api/ng/function/angular.forEach)